packages/dom/src/index.js

```diff
@@ -1,14 +1,9 @@
 import serialize from './serialize-dom';
 
 /* istanbul ignore next */
-// works around instances where the context has an incorrect global scope
-// https://github.com/mozilla/geckodriver/issues/1798
-try {
-  if (globalThis !== window) {
-    window.PercyDOM = { serialize };
-  }
-} catch (error) {
-  // `globalThis` is probably not defined
+// works around instances where the context has an incorrect scope
+if (typeof window !== 'undefined') {
+  window.PercyDOM = exports;
```

Comment:
The test error was because the test bundles use the global `PercyDOM` to reference this module, however setting this manually to a new object caused it to lose other module properties. Setting it to `exports` will set it to be the same value as the module exports (including a `default` property).